### ISSUE TITLE
Fix `value-keyword-layout-mappings` false positives for `caption-side`

### ISSUE THREAD

**Issue by romainmenke** [stylelint/stylelint#9289]
### What minimal example or steps are needed to reproduce the bug?

```css
.foo {
  caption-side: bottom;
}
```

### What minimal configuration is needed to reproduce the bug?

```js
export default {
  rules: {
    "value-keyword-layout-mappings": "flow-relative"
  },
  languageOptions: {
		directionality: {
			"block": "top-to-bottom",
			"inline": "left-to-right"
		}
	},
};

```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0AzCEAEwA6A7DGYAhgA4AuAlhLgLQDOFs8GARhGWRALYDceAviAA0IFBQA2MAHJEucRDAAeskpKRg6dYeGpiA5ghBKSEAE5kMsFEQCu4iznwZTdmHWaOCBbCABuRcRsYGgBrGABPAHczKBpxInCIGzIaLlISClw9Oh9mHxRxCEiaUxh4yl8YHzwCfiEajHismyI9GAB5cipcdyw8bDIBqApSsEpqAIoycI9+wfmfFkKwENyMH04SGk4aNg5uH3rcAZPBkEzxTKqEdZBJFBSd0wo9AAszuYH+ObqBPlxtJBcPoAGJmNJkQx0aaSS64MjqXQvJBcABWWhEsBIWkQnlu0PCsMyZzydyIZDcZyO3hABKJ8JoQP09DIRFwUCIpigax85Up1Vw3wB-CAA)

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "latest",
  "stylelint-config-standard": "latest"
}
```

### What did you expect to happen?

I expected no error

### What actually happened?

```
2:17-23 error Expected "bottom" to be "block-end" [(value-keyword-layout-mappings)](https://stylelint.io/user-guide/rules/value-keyword-layout-mappings)
```

Proposed fix to:

```css
.foo {
  caption-side: block-end;
}
```

Note that `block-end` is not a valid value, even according to the spec.

### Do you have a proposal to fix the bug?

See:
- https://drafts.csswg.org/css-logical/#caption-side
- https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Properties/caption-side

It seems that `caption-side` doesn't support logical values (yet) in any browser.
So these transforms could break projects for our users.

With that alone in mind I suggest that we remove `caption-side` from this rule for now.

Aside from browser support the `block-*` values also aren't supported. Only `inline-*` values are specified.

**Comment by jeddy3** [comment]
> With that alone in mind I suggest that we remove caption-side from this rule for now.

Sounds good.
